RussianFilter/sections/specific.txt

```diff
@@ -225,6 +225,8 @@ primamedia.ru###banner_fs_container
 ||primamedia.ru/media/css/banner_fullscreen.css
 zhacker.net##div[style="text-align: center;"] > a > img
 0day.kiev.ua##div[style="height:200px; margin:20px;"]
+0day.kiev.ua##td[valign="top"][width="99%"] > center
```

Comment:
it is hidden by `0day.kiev.ua##div[style="height:200px; margin:20px;"]` from RU filter on my end. Please check again